pkg/cli/admin/catalog/mirror_test.go

```diff
@@ -131,7 +132,7 @@ func TestGetRegistryMapping(t *testing.T) {
 			want: map[string]string{},
 		},
 		{
-			name: "src is tagged and icsp with registy scope - skip mirror",
+			name: "src is tagged and idms with registy scope - skip mirror",
```

Comment:
Since we are still supporting icsp(although it is deprecated), preserving icsp tests would be better. Would you mind renaming `TestGetRegistryMapping` to something like `TestGetRegistryMappingICSP` and add same tests for also `IDMS`.

Reply:
fixed